packages/replay-swift/Replay/Sources/Replay/ReplayClipboardManager.swift

```diff
@@ -0,0 +1,18 @@
+import UIKit
+
+class ReplayClipboardManager {
```

Comment:
Hmm something went wrong because this shouldn't be here. It looks like you have one of my commits which is already in master here somehow.